docs/developing.md

```diff
@@ -219,10 +219,10 @@ This table describes which [`NonbondedMethod`](http://docs.openmm.org/latest/use
 
 | vdw_method | electrostatics_method | periodic | OpenMM Nonbonded method or exception | Common case |
 |------------|-----------------------|----------|--------------------------------------|---|
-| cutoff     | Coulomb               | True     | raises `IncompatibleParameterError`
-| cutoff     | Coulomb               | False    | `openmm.NonbondedForce.NoCutoff`
-| cutoff     | reaction-field        | True     | raises `SMIRNOFFSpecUnimplementedError`
-| cutoff     | reaction-field        | False    | raises `SMIRNOFFSpecError`
+| cutoff     | Coulomb               | True     | raises `.SMIRNOFFSpecUnimplementedError`
+| cutoff     | Coulomb               | False    | raises `.SMIRNOFFSpecUnimplementedError`
```

Comment:
Let's have this return the same thing as before, and print a DeprecationWarning that this will change (or return something different)